packages/@ember/-internals/glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -459,44 +459,6 @@ moduleFor(
       });
     }
 
-    ['@test should apply classes of the dasherized property name when bound property specified is true']() {
```

Comment:
We chatted a bit about this in DM's, would you mind summarizing the result of our conversation (RE: why this is fine to remove)?

Reply:
Sure thing. Basically, the exact behavior here seems to be an "accidental feature". We traced it back to when it was first implemented, a long time ago, and it popped up after a number of refactors to the ways that bindings worked. This functionality was never documented, and we haven't found any users of it in existing applications, so we believe it is safe to remove.